MyArrayList.java

```diff
@@ -1 +1,100 @@
+package com.company;
 
+public class MyArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_CAPACITY = 10;
+    private int i = 0;
+    private Object[] array = new Object[DEFAULT_CAPACITY];
+    StringBuilder stringBuilder = new StringBuilder();
+    private int size;
```

Comment:
do we need this field? maybe better use array.length?

Reply:
Yes. I want to use it, because it more short than i will write array.length 
If you don't mind, i leave so. Ok?
private int size = array.length;